CRM/Utils/File.php

```diff
@@ -596,6 +596,25 @@ public static function baseFilePath() {
     }
     return $_path;
   }
+  
```

Comment:
It's not obvious on the web, but seems like the syntax check detected a trailing whitespace on this line?